Process.c

```diff
@@ -617,6 +617,7 @@ int Process_compareByKey_Base(const Process* p1, const Process* p2, ProcessField
    case USER:
       return SPACESHIP_NULLSTR(p1->user, p2->user);
    default:
+      assert(0); /* should never be reached */
```

Comment:
Maybe use comma operator to provide some meaningful message in case of assertion failure.

Reply:
added